www/includes/easyparliament/templates/html/mp/register.php

```diff
@@ -0,0 +1,94 @@
+<?php
+include_once INCLUDESPATH . "easyparliament/templates/html/mp/header.php";
+?>
+
+<div class="full-page">
+    <div class="full-page__row">
+        <div class="full-page__unit">
+            <?php include '_person_navigation.php'; ?>
+        </div>
+        <div class="person-panels">
+            <div class="sidebar__unit in-page-nav">
+                <div>
+                    <h3 class="browse-content"><?= gettext('Browse content') ?></h3>
+                    <ul>
+                      <li><a href="https://www.mysociety.org/2024/01/17/improving-the-register-of-mps-interests/">Download a spreadsheet</a></li>
+                      <?php include '_featured_content.php'; ?>
+                      <?php include '_donation.php'; ?>
+                </div>
+            </div>
+
+            <div class="primary-content__unit">
+
+              <?php if ($party == 'Sinn Féin' && in_array(HOUSE_TYPE_COMMONS, $houses)): ?>
```

Comment:
Presumably this block isn't needed, because there won't be a register for sinn fein or newly elected MPs.

Reply:
Yep, out.